crates/configuration/src/version3/mod.rs

```diff
@@ -30,7 +30,6 @@ pub const DEFAULT_CONNECTION_URI_VARIABLE: &str = "CONNECTION_URI";
 pub struct RawConfiguration {
     // Connection string for a Postgres-compatible database
     pub connection_uri: ConnectionUri,
-    #[serde(skip_serializing_if = "PoolSettings::is_default")]
```

Comment:
I assume is_default only exists for this particular setting, so if you'd like to remove it as well, you can.